packages/pocket-ic/src/nonblocking.rs

```diff
@@ -1529,7 +1529,7 @@ impl PocketIc {
         result.into()
     }
 
-    pub(crate) async fn update_call_with_effective_principal(
+    pub async fn update_call_with_effective_principal(
```

Comment:
Should we then maybe make the corresponding function in lib.rs public, too, and update the changelog?

Reply:
Done